engine/src/main/java/org/terasology/rendering/dag/stateChanges/BindFBO.java

```diff
@@ -28,33 +28,35 @@
     private static final Integer DEFAULT_FRAME_BUFFER_ID = 0;
     // TODO: add necessary checks for ensuring generating FBO with the name "display" is not possible.
     private static final String DEFAULT_FRAME_BUFFER_NAME = "display";
-    private static BindFBO defaultInstance = new BindFBO();
+    private static FrameBuffersManager frameBuffersManager;
+    private static BindFBO defaultInstance = new BindFBO(DEFAULT_FRAME_BUFFER_NAME);
     private String fboName;
-    private FrameBuffersManager frameBuffersManager;
+
     private int fboId;
     private BindFBOTask task;
 
-    public BindFBO(String fboName, FrameBuffersManager frameBuffersManager) {
+    public BindFBO(String fboName) {
         this.fboName = fboName;
-        this.frameBuffersManager = frameBuffersManager;
-        fboId = frameBuffersManager.getFBO(fboName).fboId;
+        if (fboName.equals(DEFAULT_FRAME_BUFFER_NAME)) {
```

Comment:
I think the most likely case is that fboName is not the default frame buffer. But it's a very minor issue, so let's leave it as it is.